### ISSUE TITLE
Fail if applying manifest fails

### ISSUE THREAD

**Issue by mikkeloscar** [zalando-incubator/cluster-lifecycle-manager#78]
In case applying a manifest fails, the CLM will just log an error and continue.

```
time="2018-08-10T14:40:50Z" level=error msg="Error applying template template: /home/master/workspace/ubernetes-on-aws-e2e_master-5FUAFYO2STDSYHIB2UTPTBMD2CFAR7XTT5L5MRNTC4AMDBO42YQA/head/cluster/manifests/kube-downscaler/deployment.yaml:1: function "neq" not defined" 
```

I think we added this graceful error handling in the past to work around some kubectl issues. But it probably probably makes sense to fail the provisioning if this happens.

**Comment by hjacobs** [comment]
Yes, that looks like a bug to me --- at least it's very surprising (I think my change was applied, but it was not).

**Comment by aermakov-zalando** [comment]
Fixed in #91.
